notebooks/nzh-20230417-00-embeddings-classifier.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>It's usually easiest to do this kind of thing by creating a dataframe, and then by using <code>pd.write_csv.</code> I would do something where I zip the probabilities and the classes together, and then create a dictionary from that zipped list.</p><p> </p><p>Something along these lines</p><p> </p><p><code></code><code></code>python</p><p>res = []</p><p>for row in rows:</p><p>    assert len(species) == len(predictions), "this only works when the two lists are the same dim"</p><p>    d = dict(*zip(species, predictions))</p><p>    d["row_id"] = row</p><p>    res.append(d)</p><p>res_df = pd.DataFrame(res)</p><p><code></code>`</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dsgt-birdclef/birdclef-2023/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>